server/names/names_test.go

```diff
@@ -54,8 +55,8 @@ func TestResourceNames(t *testing.T) {
 			name:   "apis",
 			regexp: ApisRegexp(),
 			pass: []string{
-				"projects/google/apis",
-				"projects/-/apis",
+				"projects/google/locations/LOCATION/apis",
```

Comment:
Should this be `global`?
Here and everywhere below.

Reply:
This package is already generalized to support arbitrary locations (and confirmed with this test) -- and since I expect that we'll want to support non-global locations, I'd like to keep this.